pepc.html

```diff
@@ -8,14 +8,48 @@
     <link rel="shortcut icon" href="favicon.ico" />
     <link rel="apple-touch-icon" href="app-icon.png" />
     <link rel="stylesheet" href="style.css" />
+    <script src="pepc.js"></script>
   </head>
   <body>  
     <div class="before"></div>
     <div class="content">      
-      <div><permission id="camera" type="camera"></permission></div>
-      <div><permission id="microphone" type="microphone"></permission></div>
-      <div><permission id="camera-microphone" type="camera microphone"></permission></div>
-      <div><permission id="geolocation" type="geolocation"></permission></div>
+      <span class="permission-element-box">
+        <div><permission id="geolocation" type="geolocation"></permission></div>
+        <div class="access-status">
+          Access status: <span id="geolocation-access-status">⚫️ unknown</span>

```

Comment:
It took me a moment to figure out this was about access status for the current page. Could we make that clear?
```suggestion
          Page access status: <span id="geolocation-access-status">⚫️ unknown</span>
```

Reply:
I'm hesitant to call it "page" permission status, as many other permissions also influence this on most browsers. E.g. it will only say GRANTED on Chrome MacOS geolocation iff

1. Device level permissions for location are granted
2. App level permissions for location are granted
3. Site level permissions for location are granted